src/components/cloudAppMock.tsx

```diff
@@ -110,7 +114,13 @@ export function CloudAppMock({ children, initialValues, authToken, teamName }: P
             signal: apiRequestAbortControllers[id].signal,
           });
 
-          const responseBody = await response.json();
+          let responseBody;
+
+          try {
```

Comment:
@jackcloudquery I'm open to other solutions here, but the PATCH 204 response fails on the initial logic

Reply:
I refactored it a bit to avoid unnecessary console log